aeronautical/asterix/cat_000.ksy

```diff
@@ -0,0 +1,9 @@
+meta:
+  id: cat_000
+  license: GPL-3.0-only
+  endian: be
+
+seq:
+  - id: bytes
+    type: u1
+    repeat: eos
```

Comment:
Why not `size-eos: true`? What's the value in having a true array of bytes here?

Reply:
Is there any drawbacks with this? Or a performance issue?. I opted to leave this way, because during testings I could change u1 with other types or integers changing just type.